arangod/IResearch/IResearchLink.cpp

```diff
@@ -1364,17 +1381,17 @@ arangodb::Result IResearchLink::initDataStore(InitCallback const& initCallback,
       auto* link = asyncSelf->get();
 
       if (!link) {
-        return arangodb::Result(); // link no longer in recovery state, i.e. during recovery it was created and later dropped
+        return {}; // link no longer in recovery state, i.e. during recovery it was created and later dropped
       }
 
       // before commit ensure that the WAL 'Flush' marker for the opened writer
       // was seen, otherwise this indicates a lost WAL tail during recovery
       // i.e. dataStore is ahead of the WAL
       if (RecoveryState::AFTER_CHECKPOINT != link->_dataStore._recovery) {
-        return arangodb::Result( // result
-          TRI_ERROR_INTERNAL, // code
-          std::string("failed to find checkpoint after finishing recovery of arangosearch link '") + std::to_string(link->id()) + "'"
-        );
+        LOG_TOPIC("31zv1", ERR, arangodb::iresearch::TOPIC)
```

Comment:
`31zv1` is an invalid log id. The rules are:
* 5 digits
* from the range `[0-9a-f]`
* globally unique